lib/playground.dart

```diff
@@ -76,16 +76,12 @@ class Playground implements GistContainer, GistController {
   ModuleManager modules = new ModuleManager();
 
   Playground() {
-    TabController controller = new TabController();
+    sourceTabController = new TabController();
```

Comment:
I think some of these changes are from a PR I've since landed. Can you merge this PR to master before committing?